src/main/java/com/linkedin/xinfra/monitor/services/ConsumeService.java

```diff
@@ -242,17 +248,18 @@ public synchronized void start() {
   @Override
   public synchronized void stop() {
     if (_running.compareAndSet(true, false)) {
-      try {
-        _baseConsumer.close();
-      } catch (Exception e) {
-        LOG.warn(_name + "/ConsumeService while trying to close consumer.", e);
-      }
-      LOG.info("{}/ConsumeService stopped.", _name);
+      LOG.info("{}/ConsumeService stopping.", _name);
     }
   }
 
   @Override
   public void awaitShutdown(long timeout, TimeUnit unit) {
+    LOG.info("{}/ConsumeService shutdown awaiting…", _name);
+    try {
+      _consumeThread.join(unit.toMillis(timeout));
+    } catch (InterruptedException e) {
+      LOG.error(_name + "/ConsumeService interrupted", e);
```

Comment:
This silently swallows the error here.  Is this `awaitShutdown` only invoked when the whole service is shutdown?

If not, we should either
1. Let the caller handle this or
2. retry and throw a runtime exception

Reply:
I think it's ok, because: 
1. exception is logged
2. that's [the way it's handled](https://github.com/linkedin/kafka-monitor/search?q=awaitShutdown+InterruptedException) in this project.

We could implement your suggestions but for consistency IMO, it should be:
1. kept the same way or
2. refactored across the project

WDYT?